server/routerlicious/api-report/protocol-base.api.md

```diff
@@ -106,23 +130,42 @@ export function isSystemMessage(message: ISequencedDocumentMessage): boolean;
 export function mergeAppAndProtocolTree(appSummaryTree: ITree_2, protocolTree: ITree_2): ICreateTreeEntry[];
 
 // @public
-export class ProtocolOpHandler {
+export class ProtocolOpHandler implements IProtocolHandler {
     constructor(minimumSequenceNumber: number, sequenceNumber: number, term: number | undefined, members: [string, ISequencedClient][], proposals: [number, ISequencedProposal, string[]][], values: [string, ICommittedProposal][], sendProposal: (key: string, value: any) => number);
     // (undocumented)
+    get attributes(): IDocumentAttributes;
+    // (undocumented)
     close(): void;
     getProtocolState(): IScribeProtocolState;
     // (undocumented)
     minimumSequenceNumber: number;
     // (undocumented)
     processMessage(message: ISequencedDocumentMessage, local: boolean): IProcessMessageResult;
     // (undocumented)
-    readonly quorum: Quorum;
+    get quorum(): Quorum;
+    // (undocumented)
+    readonly _quorum: Quorum;
     // (undocumented)
     sequenceNumber: number;
     // (undocumented)
+    setConnectionState(connected: boolean, clientId: string | undefined): void;
+    // (undocumented)
+    snapshot(): IQuorumSnapshot;
+    // (undocumented)
     readonly term: number;
 }
 
+// @public (undocumented)
+export class ProtocolOpHandlerWithClientValidation extends ProtocolOpHandler {
```

Comment:
Maybe this one should be called ProtocolOpHandler and above should be ProtocolOpHandlerEmptyQuorum?
Not sure